src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const tables = table.children;
```

Comment:
The variable name 'tables' might be misleading since it suggests multiple tables, but it actually refers to the children of a single table element. Consider renaming it to something more descriptive, like 'tableSections'.